evansmusomi/react-native-102
# Adds Auto Sign In Functionality

*What this does*
- Stores auth token in async storage
- Auto Signs In user if token is found in async storage

*How to verify*
- Log in to the app
- Close the app
- Open the app again
- You should be automatically signed in and directed to the Find Places screen
